Changes aimed at AD's review of the formal appeal (#533)
oauth-wg/oauth-selective-disclosure-jwt: draft-ietf-oauth-selective-disclosure-jwt.md

Key Binding:
:  Ability of the Holder to prove possession of an SD-JWT by proving
  control over a private key during the presentation. When utilizing Key Binding, an SD-JWT contains
  the public key corresponding to the private key controlled by the Holder (or a reference to this public key).

Key Binding JWT (KB-JWT):
:  A Key Binding JWT is said to "be tied to" a particular SD-JWT when its payload

>   includes a hash of the SD-JWT in its `sd_hash` claim.A JWT for proving Key Binding

Author: "One nit: on the Key Binding JWT def: You need a space between the sentences." 
thanks @debcooley 
```suggestion
  includes a hash of the SD-JWT in its `sd_hash` claim. A JWT for proving Key Binding
```

Reviewer: perfect.  TY

> :  A Key Binding JWT is said to "be tied to" a particular SD-JWT when its payload
>   includes a hash of the SD-JWT in its `sd_hash` claim. A JWT for proving Key Binding
>   as defined in (#kb-jwt).

Reviewer 2: but what binds KB JWT to SD-JWT is the fact that KB JWT's payload is signed using the key included in the SD-JWT payload.

```suggestion
:  A Key Binding JWT is said to "be tied to" a particular SD-JWT when its payload
  is signed using the key included in the SD-JWT payload, and also contains
  a hash of the SD-JWT in its `sd_hash` claim. A JWT for proving Key Binding
  as defined in (#kb-jwt).
```

<Issuer-signed JWT>~<Disclosure 1>~<Disclosure N>~<KB-JWT>
```

As an alternative illustration of the SD-JWT format, ABNF [@?RFC5234] for the

> SD-JWT, SD-JWT+KB, and various constituent parts is provided here (for those who celebrate):

Reviewer 2: ```suggestion
SD-JWT, SD-JWT+KB, and various constituent parts is provided here:
```

Author: I'm rather attached to that text

Author: especially around this time of year
